config.ru

```diff
@@ -0,0 +1,93 @@
+#\ -O Threads=1:1 -p 3000
+
+require 'bundler/setup'
+require 'rails'
+require 'rails/all'
+require 'solidus'
+
+module RailsApp
+  class Application < Rails::Application
+    config.root                                       = __dir__
+    config.cache_classes                              = true
+    config.eager_load                                 = false
+    config.public_file_server.enabled                 = true
+    config.public_file_server.headers                 = { 'Cache-Control' => 'public, max-age=3600' }
+    config.consider_all_requests_local                = true
+    config.action_controller.perform_caching          = false
+    config.action_dispatch.show_exceptions            = false
+    config.action_controller.allow_forgery_protection = false
+    config.active_support.deprecation                 = :stderr
+    config.secret_key_base                            = '49837489qkuweoiuoqwe'
+
+    config.logger = ActiveSupport::Logger.new(STDOUT)
+
+    config.middleware.delete Rack::Lock
+    config.middleware.delete ActionDispatch::Flash
```

Comment:
I think we should leave this in. If we want to use this a as an easy way to review PRs (instead of generating a sandbox) we want to have flash messages.